apply/default_applier.go

```diff
@@ -171,22 +181,24 @@ func (c *DefaultApplier) diff() (todoList []ActionName, err error) {
 	}
 	c.MastersToJoin, c.MastersToDelete = utils.GetDiffHosts(c.ClusterCurrent.Spec.Masters, c.ClusterDesired.Spec.Masters)
 	c.NodesToJoin, c.NodesToDelete = utils.GetDiffHosts(c.ClusterCurrent.Spec.Nodes, c.ClusterDesired.Spec.Nodes)
-	todoList = append(todoList, Mount)
+	todoList = append(todoList, MountImage)
+	todoList = append(todoList, MountRootfs)
 	if c.MastersToJoin != nil || c.MastersToDelete != nil {
 		todoList = append(todoList, ApplyMasters)
 	}
 	if c.NodesToJoin != nil || c.NodesToDelete != nil {
 		todoList = append(todoList, ApplyNodes)
 	}
 
-	// if only contains PullIfNotExist and Mount, we do nothing
-	if len(todoList) == 2 {
-		todoList = append(todoList, CNI)
-		return todoList, nil
+	// if only contains PullIfNotExist and MountImage MountRootfs , we do nothing
+	if len(todoList) == 3 {
```

Comment:
len == 3 is not friendly, we must apply Guest!

Reply:
ok . I fix .
1. check todo list has ApplyMasters, ApplyNodes, Upgrade so add UnMountImage , otherwise clean the todolist 
2. add Guest process any where